modules/formulize/class/data.php

```diff
@@ -49,14 +50,22 @@ function formulizeDataHandler($fid){
 			$this->fid = false;
 		}
 		
-		//set the avaiable metadata fields to a global
-		$this->metadataFields = array("ENTRY_ID", 
-		  						"CREATION_DATETIME",
-		   						"CREATION_UID",
-		   						"CREATOR_EMAIL",
-		  						"MOD_DATETIME",
-		   						"MOD_UID",
-		   						"USER_VIEWEMAIL");  
+		//set the available metadata fields to a global

```

Comment:
We can get rid of user_viewemail, I think that was put in here by mistake originally.